apps/web/components/Embed.tsx

```diff
@@ -710,13 +750,13 @@ const EmbedTypeCodeAndPreviewDialogContent = ({
       height: "100%",
     },
     theme: Theme.auto,
+    layout: BookerLayouts.MONTH_VIEW,
```

Comment:
Moved it along with theme because they are both ui config and makes sense to be together.